src/servers/Scripts/quest/ManSea001.cpp

```diff
@@ -27,102 +32,98 @@ class ManSea001 : public EventScript
 
    void Scene00000( Entity::Player& player )
    {
-      auto callback = [this]( Entity::Player& player, const Event::SceneResult& result )
+      player.playScene( m_id, 0, 8192,
+         [&]( Entity::Player& player, const Event::SceneResult& result )
       {
-         if( result.param2 == 1 )
+         if ( result.param2 == 1 )
          {
             player.setOpeningSequence( 2 );
             Scene00001( player );
          }
-      };
-
-      player.playScene( getId(), 0, HIDE_HOTBAR, 0, 0, callback );
+      } );
    }
 
    void Scene00001( Entity::Player& player )
    {
-      player.playSceneChain( getId(), 1, DISABLE_SKIP | HIDE_HOTBAR | SET_BASE, bindScene( &ManSea001::Scene00002 ) );
+      player.playSceneChain( m_id, 1, DISABLE_SKIP | HIDE_HOTBAR | SET_BASE, bindScene( &ManSea001::Scene00002 ) );
```

Comment:
anything that needs the script id should call `getId` not use `m_id` because we can change the behavior of the getter easily but not the variable

Reply:
Every playSceneChain and playScene need getId() or only playSceneChain? (In the first case, I need to modify a lot of scripts xD)

Reply:
ctrl shift h
m_id, 
getId(),